package/MDAnalysis/analysis/data/filenames.py

```diff
@@ -66,17 +75,28 @@
       import numpy as np
       import matplotlib.pyplot as plt
       from MDAnalysis.analysis.data.filenames import Janin_ref
-      X, Y = np.meshgrid(np.arange(-180, 180, 4), np.arange(-180, 180, 4))
+      X, Y = np.meshgrid(np.arange(-180, 180, 6), np.arange(-180, 180, 6))
```

Comment:
I looked closely at the Janin plots in https://docs.mdanalysis.org/stable/documentation_pages/analysis/dihedrals.html?highlight=ramachandran#figure-janin and the code https://github.com/MDAnalysis/mdanalysis/blob/9d090e695fae02f40328942648bb06a5e3b6553b/package/MDAnalysis/analysis/dihedrals.py#L663-L667 and realized that we plot the chi angles from 0º to 360º. Could you please adjust the example and your plot accordingly?